1_fetch/src/get_sb_data.R

```diff
@@ -214,6 +214,26 @@ download_children <-function(sites, sb_table_reduced, dldir, workdir, outdir, ou
   return(return_df)
 }
 
+calc_avg_dams <- function(sb_data) {
+  sb_data <- read_csv(sb_data, show_col_types = FALSE) %>%
+    suppressMessages()
+  dams <- sb_data %>%
+    select(COMID, contains("NDAMS"), contains("STORAGE"), contains("MAJOR")) %>%
```

Comment:
```suggestion
    select(COMID, contains("NDAMS"), contains("NID"), contains("STORAGE"), contains("MAJOR")) %>%
```

Reply:
Not sure if anything else will need to change with this additional variable

Reply:
Nevermind, Charlie noted that STORAGE captures both NID_STORAGE and TOTAL_STORAGE